api/proto/job/job.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+option go_package = "teleport-job-worker/api/proto/job";
+package job;
+
+message StartRequest {
+  job.Command command = 1;
+  job.Resources resource_limits = 2;
```

Comment:
user-specified resource limits is out of scope - you can hardcode some set of resource limits instead of letting users choose for each job.

Reply:
@GavinFrazar Not out of scope for this challenge:

> Add resource control for CPU, Memory and Disk IO **per job** using cgroups.